includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1482,54 +1421,72 @@ protected function build_display_items( $itemized_display_items = false ) {
 				$subtotal      += $cart_item['line_subtotal'];
 				$quantity_label = 1 < $cart_item['quantity'] ? ' (x' . $cart_item['quantity'] . ')' : '';
 
-				$product_name = $cart_item['data']->get_name();
-
 				$item = [
-					'label'  => $product_name . $quantity_label,
+					'label'  => $cart_item['data']->get_name() . $quantity_label,
 					'amount' => WC_Stripe_Helper::get_stripe_amount( $amount ),
 				];
 
 				$items[] = $item;
```

Comment:
Feel free to ignore as this is another tiny suggestion not related to your code:

We can remove that extra `$item` definition without sacrificing readability with:

```php
$items[] = [
	'label'  => $product_name . $quantity_label,
	'amount' => WC_Stripe_Helper::get_stripe_amount( $amount ),
];
```

Reply:
done 73e1c77